adapters/kafkastreamer/go.mod

```diff
@@ -1,10 +1,10 @@
-module github.com/luno/workflow/kafkastreamer
+module github.com/luno/workflow/adapters/kafkastreamer
 
 go 1.22.3
 
 require (
 	github.com/luno/jettison v0.0.0-20240625085333-8727b580c646
-	github.com/luno/workflow v0.1.1
```

Comment:
We could use replace directives to always just use the local latest version? Then we wouldn't need to update this all the time? (same with other modules)

Reply:
Cool will try it. I was thinking the relative path would break things but then realised go mods us the mod name to position deoendacies in a consistent location so this would always be relative on all machines. TIL.